flocker/node/test/test_loop.py

```diff
@@ -692,7 +700,8 @@ def test_connected(self):
         is passed to the cluster status FSM.
         """
         service = AgentLoopService(
-            reactor=None, deployer=object(), host=u"example.com", port=1234)
+            reactor=None, deployer=object(), host=u"example.com", port=1234,
+            context_factory=ClientContextFactory())
```

Comment:
Maybe these tests should use the existing `_build_service` helper so this fix doesn't have to be repeated so many times.